i18n/es/docusaurus-plugin-content-docs/current/basic_applications/emojis.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 40
+---
+
+# 🟢 Encontrando Emojis
+
+🤔 ¿Alguna vez necesitaste el **emoji perfecto** para un tweet o publicación?
+
+A veces es difícil recordar cuál de los miles de emojis sería el mejor. ¡Afortunadamente, la inteligencia artificial puede ayudar!
+
+Como ejemplo, intentemos encontrar los mejores emojis para agregar a este [this](https://twitter.com/learnprompting/status/1631786358330085382) tweet:
```

Comment:
```suggestion
Como ejemplo, intentemos encontrar los mejores emojis para agregar a este [tweet](https://twitter.com/learnprompting/status/1631786358330085382):
```
Creo que queda mejor asi, y de paso sacamos el 'this' (rima y todo)

Reply:
jaja si